src/components/dashboard/Dashboard.js

```diff
@@ -133,7 +126,6 @@ const Dashboard = props => {
     height: headerAvatarAnimValue,
     width: headerAvatarAnimValue,
     top: 0,
-    left: headerAvatarLeftAnimValue,
```

Comment:
Why are you removing this?

Reply:
if i use it, have this problem
![Screen Shot 2020-02-11 at 2 32 46 PM](https://user-images.githubusercontent.com/59056508/74262405-8fa93000-4cdb-11ea-907f-7b4de9c208de.png)